benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_vdbench_logs/analyze_vdbench_logs.py

```diff
@@ -1,28 +1,30 @@
+
 import csv
 import os
 import pandas as pd
 
 # display df
 from IPython.display import display
+from typeguard import typechecked
 
 # display bokeh
 from bokeh.plotting import figure, show
 from bokeh.io import output_notebook, output_file
-from bokeh.models import HoverTool
-
-from typeguard import typechecked
+from bokeh.models import HoverTool, Label, ColumnDataSource, LabelSet, Circle, NumeralTickFormatter
 
 from benchmark_runner.jupyterlab.templates.logs_operations.logs_operations import LogsOperations
+from benchmark_runner.jupyterlab.templates.analyze_workloads.analyze_workloads_operations import AnalyzeWorkloadOperations
 
 
-class AnalyzeVdbenchLogs:
+class AnalyzeVdbenchLogs(AnalyzeWorkloadOperations):
     """
     This class analyzes Vdbench logs
     """
     def __init__(self, s3_logs_url: str):
+        super().__init__()
```

Comment:
Remove if you are deleting above extended class

Reply:
I am using the init parameters:
self.clear_output()
self.width = 1200
self.height = 600